source/jsonnet/england-wales/lib/placeholders.libsonnet

```diff
@@ -174,6 +174,21 @@ local firstPersonNamePossessiveForList(listName) = {
       },
     }],
   },
+  otherAddress: {
```

Comment:
It would be a little clearer if this was named `otherHouseholdAddress`.

Reply:
Actually, this should probably just be inlined in the block definition, rather than in this file, as it's only used in one place.

Reply:
I've moved it into the block and renamed it, thanks!

Reply:
I'm not sure if you want it even more inlined than this?